vcpkg/users/assetcaching.md

```diff
@@ -1,53 +1,82 @@
 ---
-title: Asset Caching
-description: Use vcpkg to mirror sources and tools internally to guarantee build reliability.
-ms.date: 11/30/2022
+title: Asset caching with vcpkg
+description: Use asset caching with vcpkg to mirror your asset's downloads to improve build reliability.
+author: vicroms
+ms.author: viromer
+ms.prod: vcpkg
+ms.topic: conceptual
+ms.date: 9/18/2023
 ---
-# Asset caching
+# Asset caching with vcpkg
```

Comment:
I feel like 'with vcpkg' is kind of implied but :shrug:

Reply:
Three-letters: SEO